glide.yaml

```diff
@@ -1,25 +1,24 @@
 package: github.com/openshift/cluster-machine-approver
 import:
 - package: k8s.io/api
-  version: kubernetes-1.11.1
+  version: kubernetes-1.14.0
   subpackages:
   - certificates/v1beta1
   - core/v1
 - package: k8s.io/apimachinery
-  version: kubernetes-1.11.1
+  version: kubernetes-1.14.0
   subpackages:
   - pkg/fields
   - pkg/util/runtime
   - pkg/util/wait
 - package: k8s.io/client-go
-  version: kubernetes-1.11.1
+  version: kubernetes-1.14.0
   subpackages:
   - kubernetes
   - tools/cache
   - tools/clientcmd
   - util/workqueue
-- package: github.com/golang/glog
 - package: github.com/openshift/cluster-api
-  version: release-4.0
+  version: openshift-4.2-cluster-api-0.0.0-alpha.4
```

Comment:
@enj @mrogers950 FWIU this branch has updates/fixes we want/need?

Reply:
We want to stay in sync with the branch that lines up with the release of OpenShift we are shipping.  I am OK if we say `master` is good enough for the cluster API code (it probably is).  The kube code is the special unicorn that we need to pin to a specific release branch.

Reply:
I've changed to master